### PR TITLE
Initial checkin for trigger metrics

### PR DESCRIPTION
**Pull request by xrajesh**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,28 @@
+approvers:
+  - jlpadilla
+  - xrajesh
+  - SherinV
+  - zlayne
+  - arewm
```
Comment (zlayne) [inline OWNERS:6]: I think we are just doing squad members now for approvers

File: build/Configfile [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Only use git commands if it exists
+ifdef GIT
+VCS_URL ?=$(shell git config --get remote.origin.url)
+IMAGE_VERSION :=$(shell git rev-parse --short HEAD)
+endif
+
+IMAGE_NAME =$(shell cat COMPONENT_NAME)
+IMAGE_DISPLAY_NAME =Insights Ingestor
```
Comment (zlayne) [inline build/Configfile:8]: insights ingestor? or insights-metrics?

File: pkg/collectors/builder.go [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+	return collectors
+}
+
+var availableCollectors = map[string]func(f *Builder) *collector.Collector{
+	"policyreports": func(b *Builder) *collector.Collector { return b.buildPolicyReportCollector() },
+}
+
+func (b *Builder) buildPolicyReportCollector() *collector.Collector {
+	config, err := clientcmd.BuildConfigFromFlags(b.apiserver, b.kubeconfig)
```
Comment (zlayne) [inline pkg/collectors/builder.go:112]: think you have this same func in this file: `policyreport_listwatch`

File: OWNERS (hunk 1 above)
Comment (xrajesh) [reply OWNERS]: Fixed it

File: build/Configfile (hunk 2 above)
Comment (xrajesh) [reply build/Configfile:8]: Fixed it

**Review by zlayne — APPROVED** [review]

**Pull request merged**